deployment/postgres-init-db/sql/tables/sequencing/expansion_rule.sql

```diff
@@ -5,7 +5,7 @@ create table sequencing.expansion_rule (
   activity_type text not null,
   expansion_logic text not null,
 
-  authoring_command_dict_id integer,
+  parcel_id integer not null,
```

Comment:
I am sorry, I don't follow the question. A `Parcel` is required for each `Exapansion Rule`. Each `Parcel` will contain a `command dictionary`, optional `channel dictionary`, optional` parameter dictionary`, and an optional `Seq adaptation JavaScript` file. 

Within the UI, you'll only be able to select Parcels. We'll then automatically extract the necessary data from the chosen Parcel.

Reply:
So currently, `expansion_rule`s could have a `null` `command_dictionary_id`, meaning they were not required to be associated with a command dictionary. In this PR, `expansion_rules`s _must_ be associated with a parcel, which _must_ be associated with a `command_dictionary_id`, so `expansion_rule`s _must_ be associated with a command dictionary (through the `rule`'s parcel).

Additionally, the case of "`expansion_rule` with `null command_dict_id`" will likely cause issues with your migration. The easiest way to solve this would be to drop any `expansion_rule`s where `authoring_command_dict_id is null`.

Reply:
Got it, I added this to the `up.sql`  in the section:

```
--------------------------------------
---- Modify the Expansion Rule Table
--------------------------------------
```

add this
```
--- drop rule with no command dictionary
delete from sequencing.expansion_rule er
where authoring_command_dict_id is null;
```

Reply:
I'll resolve this once I've had a chance to review the `up.sql`, but that change looks sufficient.